asn1c/asn1c.c

```diff
@@ -179,6 +186,11 @@ main(int ac, char **av) {
         case 'S':
             skeletons_dir = optarg;
             break;
+        case 'D':
+            strncat(destdir, optarg, PATH_MAX - 1);
+            if(destdir[strlen(destdir)-1] != '/')
+                strncat(destdir, "/", PATH_MAX - 2);
```

Comment:
`(PATH_MAX - 2)` argument to `strncat()` does not make sense here.

Reply:
Ok. I misinterpreted the strncat page. I think this works better:
```c
        case 'D':
            strncat(destdir, optarg, PATH_MAX - 2); /* leave room for possible trailing '/' */
            if(destdir[strlen(destdir)-1] != '/')
                strncat(destdir, "/", PATH_MAX - 1);
```